build/dev/run_locally.sh

```diff
@@ -5,8 +5,8 @@ set -eo pipefail
 # This script will deploy a standalone DSS instance with docker-compose using
 # images from Docker Hub.
 
-if [[ -z $(command -v docker-compose) ]]; then
-  echo "docker-compose is required but not installed.  Visit https://docs.docker.com/compose/install/ to install."
+if [[ -z $(command -v docker compose) ]]; then
```

Comment:
The "compose" doesn't do anything here since just "docker" is now the argument; the path to `docker` will be returned, so let's make it clear what's being checked:

```suggestion
if [[ -z $(command -v docker) ]]; then
```